### PR TITLE
CXF-83183 Adding Service Token module and metal-2-fabric example with Metal Provider

### PR DESCRIPTION
**Pull request by srushti-patl**
Added:
- Service Token Module
- Metal-2-Fabric Connection examples with Metal Provider

### CODE REVIEW

**Review by thogarty — CHANGES_REQUESTED** [review]
Small changes requested.

File: modules/port-connection/variables.tf [hunk 1]
```diff
@@ -45,18 +45,15 @@ variable "aside_port_name" {
   description = "Equinix A-Side Port Name; your tagging must match the encapsulation type of the port (DOT1Q or QINQ)"
   type        = string
 }
-
```
[the remark this answers is not in the document]
Comment (srushti-patl) [reply modules/port-connection/variables.tf:48]: No, I was just going through this file and removed it manually.

[the remark this answers is not in the document]
Comment (thogarty) [reply modules/port-connection/variables.tf:48]: All good.

**Review by thogarty — APPROVED** [review]
LGTM, shipping it.

**Pull request merged**